src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const numbers = document.querySelectorAll('.population');
+
+const list = [];
+
+numbers.forEach(item => list.push(item.innerText));
+
+const population = list.map(item => {
+  return +item.split(',').join('');
+});
+const totalPopulation = population.reduce((prev, num) => prev + num, 0);
+const avaragePopulation = totalPopulation / population.length;
+
+const totalToIncert = totalPopulation.toLocaleString('en-US');
+const avarageToIncert = avaragePopulation.toLocaleString('en-US');
+
+const elementTotal = document.querySelectorAll('.total-population');
+const elementAvarage = document.querySelectorAll('.average-population');
+
+elementTotal[0].innerHTML = totalToIncert;
+elementAvarage[0].innerHTML = avarageToIncert;
```

Comment:
If you will use `.querySelector` instead of `.querySelectorAll`, you will get the only first element. In this case, you don't use an index in elementTotal[0], because you already will have only 1 element.

Reply:
fixed